### DISCUSSION TITLE
ShapeCrawler on commerical usage

### DISCUSSION THREAD

**Discussion by ZubairImran** [ShapeCrawler/ShapeCrawler#769] [Q&A] [answered]
First of all thank you for this wonderful library. Is it ok to use ShapeCraweler in commercials software. I could see all dependencies or either MIT or MS-PL, so there should not be any problem. However need your guidance.

**Comment by AdamShakhabov** [comment, marked as answer]
Hello @ZubairImran,

thank you for your feedback👍

Yes, you are right, it can be used in commercial products.

**Reply by ZubairImran** [reply]
Thanks a lot @ashahabov for a prompt reply
